components/src/components/root/root.tsx

```diff
@@ -15,14 +18,26 @@ export class Root {
     @Event()
     rendered: EventEmitter;
 
+    @Listen('hashchange')
+    hashChangeEventHandler(event: CustomEvent) {
```

Comment:
Oh, wait -- is this needed?

Reply:
hmmmm..... yeah let me see. Maybe it was left over, when I was trying to get it working. Not sure if it does. Good catch.

Reply:
ok. removed the event handler. will merge up once everything passes.